src/node/entities.h

```diff
@@ -60,50 +60,68 @@ namespace ccf
 
   struct Tables
   {
-    // Governance tables
+    // Service tables
+
+    // Member identities
     static constexpr auto MEMBERS = "public:ccf.gov.members";
-    static constexpr auto MEMBER_ACKS = "public:ccf.gov.member_acks";
-    static constexpr auto MEMBER_CERT_DERS = "public:ccf.gov.member_cert_ders";
-    static constexpr auto MEMBER_DIGESTS = "public:ccf.gov.member_digests";
+    static constexpr auto MEMBER_ACKS = "public:ccf.gov.members.acks";
+    static constexpr auto MEMBER_CERT_DERS =
+      "public:ccf.internal.members.certs_der";
+    static constexpr auto MEMBER_DIGESTS =
+      "public:ccf.internal.members.digests";
+
+    // User identities
     static constexpr auto USERS = "public:ccf.gov.users";
-    static constexpr auto USER_CERT_DERS = "public:ccf.gov.user_cert_ders";
-    static constexpr auto USER_DIGESTS = "public:ccf.gov.user_digests";
+    static constexpr auto USER_CERT_DERS =
+      "public:ccf.internal.users.certs_der";
+    static constexpr auto USER_DIGESTS = "public:internal.gov.users.digests";
```

Comment:
This doesn't match the naming conventions of other tables.

```suggestion
    static constexpr auto USER_DIGESTS = "public:ccf.internal.users.digests";
```